net-misc/telnet-bsd/telnet-bsd-1.2-r2.ebuild

```diff
@@ -38,12 +41,10 @@ src_configure() {
 }
 
```

Comment:
Would be nice if append-libs worked instead but not tried.

Reply:
Yeah looks like append-libs was already there at some point, but removed due to bug https://bugs.gentoo.org/559154

Reply:
eh, okay, let's not worry about it. add the bug reference above it though. Also, why are we calling emake in src_configure!?

Reply:
uhhh I removed that atrocity but somehow got lost while amending/rebasing